pkg/operators/wasm/testdata/badguest/program.go

```diff
@@ -40,8 +40,8 @@ const (
 	subscriptionTypePacket subscriptionType = 3
 )
 
-// Invalid string: Too big (4GB) and offset too big (64MB)
-const invalidStrPtr uint64 = uint64(1024 * 1024 << 32)
+// Invalid string: Too big (3GB)
+const invalidStrPtr uint64 = uint64(1024 * 1024 * 1024 * 3 << 32)
```

Comment:
@eiffel-fl Since modified this line last: Is this change correct with the intentions you had for the test?

Before it was 1MB with offset 0 (Comment was wrong). And the WASM layers compiled with GO 1.24 happily returned you that memory

Reply:
> Before it was 1MB with offset 0 (Comment was wrong). And the WASM layers compiled with GO 1.24 happily returned you that memory

Good finding. 

> Is this change correct with the intentions you had for the test?

My idea was to test something bigger than the maximum memory allowed by the gadget:

`WithMemoryLimitPages(256) // 16MB (64KB per page)`

The following should work:

```
// Invalid string: Too big (17MB)
const invalidStrPtr uint64 = uint64(1024 * 1024 *  17 << 32)
```